### ISSUE TITLE
Couldn't resolve host 'https:' error for cbk.download.casteml

### ISSUE THREAD

**Issue by yyachi** [misasa/chelyabinsk#10]
```
> system("cat ~/.orochirc")
---
uri: https://dream.misasa.okayama-u.ac.jp/demo
user: admin
password: admin
> cbk.download.casteml("20160627191317-464538")
cbk.download.casteml:26: stone # => 20160627191317-464538 
cbk.download.casteml.1:26: stone # => 20160627191317-464538 
cbk.download.casteml.1:83: url # => http://https://dream.misasa.okayama-u.ac.jp/demo/records/20160627191317-464538.pml 
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Couldn't resolve host 'https:'
> 
```

**Comment by yyachi** [comment]
```
> packageVersion('chelyabinsk')
[1] ‘1.0.6’
> library(chelyabinsk)
> system("cat ~/.orochirc")
---
uri: https://dream.misasa.okayama-u.ac.jp/pub/
#user: admin
#password: admin
> cbk.download.casteml("20160627191317-464538")
cbk.download.casteml:26: stone # => 20160627191317-464538 
cbk.download.casteml.1:26: stone # => 20160627191317-464538 
cbk.download.casteml.1:83: url # => https://dream.misasa.okayama-u.ac.jp/pub/records/20160627191317-464538.pml 
cbk.download.casteml:67: cmd # => casteml join /tmp/Rtmp9mrhxu/e36a2ac42da1982898aa3a41306b9964.pml 
[1] "/tmp/Rtmp9mrhxu/59f28b76c08eb3f2dcbe2048af5203e5.pml"
>
```

**Comment by yyachi** [comment]
fixed in version 1.0.6
